### ISSUE TITLE
Budget refill proposal preview logic error

### ISSUE THREAD

**Issue by traumschule** [Joystream/pioneer#4407]
https://pioneerapp.xyz/#/proposals/preview/344
This forum budget refill was created when the group budget was ~600 JOY. Now after execution the same proposal page uses the new budget as base and shows wrong values:

The logic error is to use "CURRENT" budget value vs at the time of proposal creation. When fixing also change the copy from "current" to "prior" or "previous".

Additionally add a "DEBT" box and reflect it in calculations to enable CM to better predict the result.

## Observed
> Forum
> AMOUNT
> 10,000

> CURRENT WG BUDGET
> 10,285.76

> EXPECTED WG BUDGET
> 20,285.76

## Expected
> AMOUNT
> 10,000

> WG DEBT
> 0

> PRIOR WG BUDGET
> 600

> EXPECTED WG BUDGET
> 10600

**Comment by dmtrjsg** [comment]
We need to be careful about it, as we should not show the budget at the moment the proposal was created, but the point when proposal was executed (or the current budget if not executed).

**Comment by traumschule** [comment]
Yes, for CM to decide it needs to be the current value. When deciding ends it has to change to the fixed value at block.
